.opencode/skills/pr-plots/SKILL.md

```
@@ -0,0 +1,90 @@
+6. **Run the script**: `python _gen_before_plots.py`.
+
+## Generating "after" plots (post-fix state)
+
+1. **Checkout the working branch** (develop or feature branch).
+2. **Run the same script** with output filenames changed to `after_<scenario>.png`.
+
+## Updating PR body
+
+1. **Write the PR body markdown** to `/tmp/pr_body.md` with `<img>` placeholder tags (empty `src=""`).
+2. **Update via gh CLI REST API** (the GitHub MCP token lacks PR write permission):
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)